### DISCUSSION TITLE
Plausible v2.1.2 + Funnels

### DISCUSSION THREAD

**Discussion by luminous8** [plausible/analytics#4639] [Self-Hosted Support] [answered]
Hi,

I have been looking for funnels, and from my understanding, they should be included in version 2.1.2, but I cannot find them.
Can you confirm whether or not they've been included?

Thanks

**Comment by ruslandoga** [comment, marked as answer]
👋 @luminous8 

AFAIK funnels are not part of CE: https://plausible.io/blog/community-edition

> We’re also keeping some of the newly released business and enterprise features ([funnels](https://plausible.io/docs/funnel-analysis) and [ecommerce revenue metrics](https://plausible.io/docs/ecommerce-revenue-tracking) at the time of being) exclusive to the business plan subscribers on our Plausible Analytics managed hosting.

**Reply by luminous8** [reply]
Oh thanks, i've missed that info
